internal/server/v1/observations/golden/bulk_point/preferred_latest.json

```diff
@@ -260,11 +319,21 @@
       "measurement_method": "BLSSeasonallyAdjusted",
       "observation_period": "P1M"
     },
+    "3981252704": {
+      "import_name": "WorldDevelopmentIndicators",
+      "provenance_url": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
+      "observation_period": "P1Y"
+    },
```

Comment:
Are any of these diffs due to the change in the logic?

Reply:
Yes,  these are from the logic change and are as expected.